vowpalwabbit/parse_args.cc

```diff
@@ -1313,14 +1288,17 @@ void parse_reductions(options_i& options, vw& all)
   reductions.push_back(OjaNewton_setup);
   // reductions.push_back(VW_CNTK::setup);
 
+  reductions.push_back(mf_setup);
```

Comment:
Is it ok for mf to move here? I assume mf requires the expanded namespaces whereas the others don't?

Reply:
From what I can tell it is okay. MF doesnt use the generate_interaction call and does it manually, so it needs the compiled set of interactions